main.go

```diff
@@ -205,6 +207,9 @@ func initLog(ctx *cli.Context) {
 	if disableLogFile {
 		log.InitLog(logLevel, log.Stdout)
 	} else {
+		dataDir := ctx.String(utils.GetFlagName(utils.DataDirFlag))
```

Comment:
Default DataDir is './Chain'.
Then default Logdir is "./Chain/Log'.
Current default log dir is "./Log"

This will break most current log processing tools.